extensions/common/sql/sql-lease/src/test/java/org/eclipse/edc/sql/lease/SqlLeaseContextTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       SAP SE - tests for bugfix (pass correct lease id for deletion)
+ *
+ */
+
+package org.eclipse.edc.sql.lease;
+
+import org.eclipse.edc.spi.monitor.Monitor;
+import org.eclipse.edc.transaction.local.LocalTransactionContext;
+import org.eclipse.edc.transaction.spi.TransactionContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.Clock;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+class SqlLeaseContextTest {
+
+    private final Monitor monitor = Mockito.mock(Monitor.class);
+    private final TransactionContext transactionContext = new LocalTransactionContext(monitor);
+    private final LeaseStatements leaseStatements = new TestEntityLeaseStatements();
+    private final Clock clock = Clock.systemUTC();
+    private final Duration leaseDuration = Duration.of(1, ChronoUnit.MINUTES);
+    private final String leaseHolder = "test-lease-holder";
+    private Connection connection;
+    private PreparedStatement statement;
+    private ResultSet resultSet;
+
+    private SqlLeaseContext sqlLeaseContext;
+
+    @BeforeEach
+    void setUp() {
+        connection = Mockito.mock(Connection.class);
+        statement = Mockito.mock(PreparedStatement.class);
+        resultSet = Mockito.mock(ResultSet.class);
+
+        sqlLeaseContext = new SqlLeaseContext(transactionContext, leaseStatements, leaseHolder, clock, leaseDuration,
+                connection);
+    }
+
+    @Test
+    void givenNoLeasePresent_WhenAcquiringLease_ShouldAcquireLease() throws SQLException {
```

Comment:
please follow our convention for test methods outlined in `docs/developer/architecture/coding-principles.md` specifically regarding test method naming. 
also, don't use upper case letters after underscores.